src/main/java/genji/Ui.java

```diff
@@ -0,0 +1,59 @@
+package genji;
+
+import genji.task.Task;
+import genji.task.TaskList;
+import java.util.Scanner;
+
+public class Ui {
+    private final String LINE = "________________________________________________________________";
+    private final Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void welcome() {
```

Comment:
Function name should start with a verb as mention in the coding standards.

**For example:**
```java
public void printWelcome() {
    showLine();
    System.out.println("Hello! I'm Genji\nWhat can I do for you?  O.o?");
    showLine();
}
```